eZ/Publish/Core/Limitation/OwnerLimitationType.php

```diff
@@ -138,10 +138,15 @@ public function evaluate(APILimitationValue $value, APIUserReference $currentUse
             );
         }
 
+        $userId = $currentUser->getUserId();
+
         /*
          * @var $object ContentInfo
          */
-        return $object->ownerId === $currentUser->getUserId();
+        $isOwner = $object->ownerId === $userId;
+        $isSelf = $object->id === $userId;
+
+        return $isOwner || $isSelf;
```

Comment:
Maybe I'd have kept the whole thing (comparisons included) on the same line. It would have been readable enough, imho.